README.md

```diff
@@ -1,25 +1,30 @@
-[![Build status](https://travis-ci.org/rust-embedded/r0.svg?branch=master)](https://travis-ci.org/rust-embedded/r0)
+# r0
+
+[![Build status](https://api.travis-ci.org/rust-embedded/r0.svg?branch=master)](https://travis-ci.org/rust-embedded/r0)
 [![crates.io](https://img.shields.io/crates/d/r0.svg)](https://crates.io/crates/r0)
 [![crates.io](https://img.shields.io/crates/v/r0.svg)](https://crates.io/crates/r0)
 
-# `r0`
+Memory initialization code **"crt0"**-like written in Rust. This crate is for bare metal systems where there is no ELF loader or OS to take care of initializing RAM for the program.
+
+This project is used by other crates, like:
```

Comment:
```suggestion
This project is meant to be used by runtime crates, like:
```

Might also be worth explicitly mentioning that this is not intended to be a user-facing crate. But this should be fine.